src/pages/Attendance.jsx

```diff
@@ -1,11 +1,220 @@
+import React from "react";
+import dayjs from "dayjs";
+import "dayjs/locale/ko";
+import { Stack, styled } from "@mui/material";
+import Chip from "@mui/material/Chip";
+import Typography from "@mui/material/Typography";
+import Checkbox from "@mui/material/Checkbox";
+import { LocalizationProvider } from "@mui/x-date-pickers/LocalizationProvider";
+import { AdapterDayjs } from "@mui/x-date-pickers/AdapterDayjs";
+import { MobileDatePicker } from "@mui/x-date-pickers/MobileDatePicker";
+import CalendarMonthIcon from "@mui/icons-material/CalendarMonth";
+import ChurchIcon from "@mui/icons-material/Church";
+import ArrowLeftIcon from "@mui/icons-material/ArrowLeft";
+import ArrowRightIcon from "@mui/icons-material/ArrowRight";
+import SettingsIcon from "@mui/icons-material/Settings";
 import Layout from "../components/Layout/Layout";
+import Title from "../components/shared/Title";
+import mcyMember from "../components/shared/member.js";
```

Comment:
```suggestion
import McyMember from "../components/shared/Member.js";
```
컴포넌트 명은 항상 대문자로 시작해야 함. 컴포넌트라면...

Reply:
위에 남겼지만 얘는 컴포넌트가 아니네...